packages/fxa-react/lib/test-utils/test.ftl

```diff
@@ -0,0 +1,11 @@
+-term = Mozilla
```

Comment:
Btw, note for l10n/future readers: these strings will _not_ make it into the l10n repo. We don't extract any strings from `fxa-react`.